packages/strapi-parts/src/Card/Card.stories.mdx

```diff
@@ -27,7 +28,18 @@ This is the doc of the `Card` component
 
 ## Card
 
-Description...
+Cards are used to gather a set of information that needs to stand out from the rest of the page. 
+
+**Best practices**
+
+- Content within the Card is mostly optional. Pick up what is needed.
+- Content can be added to the Card. Keep in mind that spacings are important in a Card component.
+- Main text is truncated at the end of the first line.
+- Prioritize content within the Card so users know exactly what are the main information.
+- If Call-to-Actions are used, they should be placed at the bottom of the Card.
```

Comment:
This has not been implemented not spec-ed

Reply:
Alright, we can remove that line then

Reply:
We would need a spec and a design for this so that we keep consistency and don't lose track of it 😊

Reply:
I think we can remove it for now, and see what usages the users request :)